### DISCUSSION TITLE
get_started.ipynb is in bundle/ but runs as if in bundle/spleen_segmentation/

### DISCUSSION THREAD

**Discussion by Leengit** [Project-MONAI/tutorials#1003] [General]
**Describe the bug**

Although `get_started.ipynb` lives in the `bundle/` directory, the path references it uses within it (e.g., `configs/train.json`) are instead relative to `bundle/spleen_segmentation/`.  This means that the notebook does not run to completion unless it is first modified.

**Expected behavior**
Perhaps move the notebook to the `bundle/spleen_segmentation/` directory.  Alternatively, change the paths such as `configs/...` to `spleen_segmentation/configs/...`.

**Environment (please complete the following information):**
 - Ubuntu 20.04.1
 - Python 3.8.10
 - MONAI 170093375ce29267e45681fcec09dfa856e1d7e7
 - CUDA 11.8.0-1 / cuDNN 8.6.0.163-1+cuda11.8
 - GPU: NVIDIA RTX A5000

**Comment by KumoLiu** [comment]
Hi @Leengit, [get_started.ipynb](https://github.com/Project-MONAI/tutorials/blob/main/bundle/get_started.ipynb) in `bundle/` directory is a guide to help get started to develop a bundle package, but not for running directly. 
If you want to download and get started with some existing models in MONAI, you can refer to the [model_zoo repo](https://github.com/Project-MONAI/model-zoo/tree/dev/models). 
You can also refer to tutorials in [this folder](https://github.com/Project-MONAI/tutorials/tree/main/model_zoo) to find how to use bundle in your own dataset or some other applications.
Hope it can help you, thanks!
